components/Books.js

```diff
@@ -268,6 +290,55 @@ const Books = () => {
             </select>
           </div>
           <div className="section">{renderContent(option)}</div>
+
+          <div className="section">
+            <span class="cdx-checkbox">
+              <input
+                id="checkbox-description-css-only-1"
+                class="cdx-checkbox__input"
+                type="checkbox"
+                aria-describedby="cdx-description-css-1"
+                onChange={(event) =>
+                  setIsEmailNotification(event.target.checked)
+                }
+                disabled={!isUserEmailable}
+                title={
+                  isUserEmailable
+                    ? ""
+                    : "No email associated with this user account"
+                }
+              />
+              <span class="cdx-checkbox__icon"></span>
+              <div
+                class="cdx-checkbox__label cdx-label"
+                title={
+                  isUserEmailable
+                    ? ""
+                    : "No email associated with this user account"
+                }
+              >
+                <label
+                  for="checkbox-description-css-only-1"
+                  class="cdx-label__label"
+                >
+                  Notify updates via e-mail
+                </label>
+              </div>
+            </span>
+
+            {isEmailNotification && (
+              <span id="cdx-description-css-1" class="cdx-label__description">
+                <p>
+                  <span class="cdx-css-icon--info-filled"></span>
+                  &nbsp; BUB2 will send an email to your email ID associated
+                  with your Wikimedia account regarding the success or failure
+                  of the upload. If no email is added to the account, email will
+                  not be sent.
```

Comment:
Since we are now checking for _emailable_ status, this line about _If no email is added to the account, email will not be sent_ can be removed I believe.